pkg/build/strategies/onbuild/onbuild.go

```diff
@@ -47,8 +47,7 @@ func New(request *api.Request) (*OnBuild, error) {
 		tar:    tar.New(),
 	}
 	// Use STI Prepare() and download the 'run' script optionally.
-	request.InstallDestination = "upload/src"
```

Comment:
See [this comment](https://github.com/openshift/source-to-image/pull/204#discussion_r31238157)